### ISSUE TITLE
Client Error: Transaction Initialize ... 401 not found

### ISSUE THREAD

**Issue by doubleprincez** [unicodeveloper/laravel-paystack#95]
I received the error below while trying to redirect through my webhook to the payment gateway.

``` "Client error: `POST http:\/\/site_url\/payment\/webhook\/transaction\/initialize` resulted in a `404 Not Found` response:\n{\n    \"message\": \"\",\n    \"exception\": \"Symfony\\\\Component\\\\HttpKernel\\\\Exception\\\\NotFoundHttpException\",\n    \"file\": \"C (truncated...)\n" ```

in payment controller...
```   
use Paystack;

 public function paymentSubmit()
    {
        try {
            return  Paystack::getAuthorizationUrl()->redirectNow();
        } catch (Exception $e) {
            return response()->json($e->getMessage());
        }
    }
```
composer information:
```
 "php": "^7.1.3",
"laravel/framework": "5.7.*",
       "unicodeveloper/laravel-paystack": "1.0.*",
```

**Comment by bere6255** [comment]
how do you dandle the redirect drop the front-end code

**Comment by doubleprincez** [comment]
I havent sent it to the front end yet, im trying to get the response first but its not redirecting to the payment gateway.....

```
        $paymentDetails = (new Paystack)->getPaymentData();
//
        $json_reply = json_decode(json_encode($paymentDetails))->data;

        $user = User::where('id', '=', auth()->id())->first();
//         document transaction
        $this->storeTransaction($user, $paymentDetails);
        return response()->json($json_reply);
```
having routes of
```
Route::post('/payment/webhook', 'CheckoutController@paymentSubmit')->name('payment.webhook');

Route::post('/payment/callback', 'CheckoutController@paymentCallback')->name('payment.callback');
```

passing the authorization url response to dd, i got
```
 GuzzleHttp\Exception\ClientException {#1138 ▼
  -request: GuzzleHttp\Psr7\Request {#1149 ▼
    -method: "POST"
    -requestTarget: null
    -uri: GuzzleHttp\Psr7\Uri {#1153 ▼
      -scheme: "http"
      -userInfo: ""
      -host: "site.dev"
      -port: null
      -path: "/payment/webhook/transaction/initialize"
      -query: ""
      -fragment: ""
    }
    -headers: array:5 [▼
      "Authorization" => array:1 [▶]
      "Content-Type" => array:1 [▼
        0 => "application/json"
      ]
      "Accept" => array:1 [▼
        0 => "application/json"
      ]
      "User-Agent" => array:1 [▼
        0 => "GuzzleHttp/6.5.1 curl/7.64.0 PHP/7.3.7"
      ]
      "Host" => array:1 [▼
        0 => "site.dev"
      ]
    ]
    -headerNames: array:5 [▼
      "authorization" => "Authorization"
      "content-type" => "Content-Type"
      "accept" => "Accept"
      "user-agent" => "User-Agent"
      "host" => "Host"
    ]
    -protocol: "1.1"
    -stream: GuzzleHttp\Psr7\Stream {#1151 ▼
      -stream: stream resource @622 ▼
        wrapper_type: "PHP"
        stream_type: "TEMP"
        mode: "w+b"
        unread_bytes: 0
        seekable: true
        uri: "php://temp"
        options: []
      }
      -size: 1774
      -seekable: true
      -readable: true
      -writable: true
      -uri: "php://temp"
      -customMetadata: []
    }
  }
  -response: GuzzleHttp\Psr7\Response {#1141 ▼
    -reasonPhrase: "Not Found"
    -statusCode: 404
    -headers: array:8 [▼
      "Date" => array:1 [▼
        0 => "Thu, 19 Mar 2020 10:17:07 GMT"
      ]
      "Server" => array:1 [▼
        0 => "Apache/2.4.39 (Win64) OpenSSL/1.1.1c PHP/7.3.7"
      ]
      "Vary" => array:1 [▼
        0 => "Authorization"
      ]
      "X-Powered-By" => array:1 [▼
        0 => "PHP/7.3.7"
      ]
      "Cache-Control" => array:1 [▼
        0 => "no-cache, private"
      ]
      "Content-Length" => array:1 [▼
        0 => "7860"
      ]
      "Connection" => array:1 [▼
        0 => "close"
      ]
      "Content-Type" => array:1 [▼
        0 => "application/json"
      ]
    ]
    -headerNames: array:8 [▼
      "date" => "Date"
      "server" => "Server"
      "vary" => "Vary"
      "x-powered-by" => "X-Powered-By"
      "cache-control" => "Cache-Control"
      "content-length" => "Content-Length"
      "connection" => "Connection"
      "content-type" => "Content-Type"
    ]
    -protocol: "1.0"
    -stream: GuzzleHttp\Psr7\Stream {#1143 ▼
      -stream: stream resource @628 ▼
        wrapper_type: "PHP"
        stream_type: "TEMP"
        mode: "w+b"
        unread_bytes: 0
        seekable: true
        uri: "php://temp"
        options: []
      }
      -size: 7860
      -seekable: true
      -readable: true
      -writable: true
      -uri: "php://temp"
      -customMetadata: []
    }
  }
  -handlerContext: []
  #message: """
    Client error: `POST http://site.dev/payment/webhook/transaction/initialize` resulted in a `404 Not Found` response:\n
    {\n
        "message": "",\n
        "exception": "Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException",\n
        "file": "C (truncated...)\n
    """
  #code: 404
  #file: "C:\xampp\htdocs\laravel_site_offline\vendor\guzzlehttp\guzzle\src\Exception\RequestException.php"
  #line: 113
  trace: {▶}
```

**Comment by doubleprincez** [comment]
This same error was gotten by someone but was closed  #62 and even [Someone](https://github.com/unicodeveloper/laravel-paystack/issues/16#issuecomment-482916688) did make mention of this but received no attention too

```"Client error: `POST http://site.dev/payment/webhook/transaction/initialize` resulted in a `404 Not Found` response:\n{\n    \"message\": \"\",\n    \"exception\": \"Symfony\\\\Component\\\\HttpKernel\\\\Exception\\\\NotFoundHttpException\",\n    \"file\": \"C (truncated...)\n",
"#0 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\guzzlehttp\\guzzle\\src\\Middleware.php(65): GuzzleHttp\\Exception\\RequestException::create(Object(GuzzleHttp\\Psr7\\Request), Object(GuzzleHttp\\Psr7\\Response))\n#1 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\guzzlehttp\\promises\\src\\Promise.php(203): GuzzleHttp\\Middleware::GuzzleHttp\\{closure}(Object(GuzzleHttp\\Psr7\\Response))\n#2 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\guzzlehttp\\promises\\src\\Promise.php(156): GuzzleHttp\\Promise\\Promise::callHandler(1, Object(GuzzleHttp\\Psr7\\Response), Array)\n#3 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\guzzlehttp\\promises\\src\\TaskQueue.php(47): GuzzleHttp\\Promise\\Promise::GuzzleHttp\\Promise\\{closure}()\n#4 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\guzzlehttp\\promises\\src\\Promise.php(246): GuzzleHttp\\Promise\\TaskQueue->run(true)\n#5 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\guzzlehttp\\promises\\src\\Promise.php(223): GuzzleHttp\\Promise\\Promise->invokeWaitFn()\n#6 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\guzzlehttp\\promises\\src\\Promise.php(267): GuzzleHttp\\Promise\\Promise->waitIfPending()\n#7 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\guzzlehttp\\promises\\src\\Promise.php(225): GuzzleHttp\\Promise\\Promise->invokeWaitList()\n#8 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\guzzlehttp\\promises\\src\\Promise.php(62): GuzzleHttp\\Promise\\Promise->waitIfPending()\n#9 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\guzzlehttp\\guzzle\\src\\Client.php(183): GuzzleHttp\\Promise\\Promise->wait()\n#10 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\guzzlehttp\\guzzle\\src\\Client.php(96): GuzzleHttp\\Client->request('post', 'http://site....', Array)\n#11 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\phemorah\\laravel-paystack\\src\\Paystack.php(164): GuzzleHttp\\Client->__call('post', Array)\n#12 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\phemorah\\laravel-paystack\\src\\Paystack.php(143): Phemorah\\Paystack\\Paystack->setHttpResponse('/transaction/in...', 'POST', Array)\n#13 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\phemorah\\laravel-paystack\\src\\Paystack.php(176): Phemorah\\Paystack\\Paystack->makePaymentRequest()\n#14 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Support\\Facades\\Facade.php(237): Phemorah\\Paystack\\Paystack->getAuthorizationUrl()\n#15 C:\\xampp\\htdocs\\laravel_site_offline\\app\\customFunctions\\PaystackFunction.php(35): Illuminate\\Support\\Facades\\Facade::__callStatic('getAuthorizatio...', Array)\n#16 [internal function]: App\\Http\\Controllers\\CheckoutController->paymentSubmit()\n#17 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Controller.php(54): call_user_func_array(Array, Array)\n#18 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\ControllerDispatcher.php(45): Illuminate\\Routing\\Controller->callAction('paymentSubmit', Array)\n#19 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Route.php(219): Illuminate\\Routing\\ControllerDispatcher->dispatch(Object(Illuminate\\Routing\\Route), Object(App\\Http\\Controllers\\CheckoutController), 'paymentSubmit')\n#20 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Route.php(176): Illuminate\\Routing\\Route->runController()\n#21 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Router.php(682): Illuminate\\Routing\\Route->run()\n#22 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Pipeline.php(30): Illuminate\\Routing\\Router->Illuminate\\Routing\\{closure}(Object(Illuminate\\Http\\Request))\n#23 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\passport\\src\\Http\\Middleware\\CreateFreshApiToken.php(49): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}(Object(Illuminate\\Http\\Request))\n#24 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(163): Laravel\\Passport\\Http\\Middleware\\CreateFreshApiToken->handle(Object(Illuminate\\Http\\Request), Object(Closure))\n#25 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))\n#26 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Middleware\\SubstituteBindings.php(41): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}(Object(Illuminate\\Http\\Request))\n#27 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(163): Illuminate\\Routing\\Middleware\\SubstituteBindings->handle(Object(Illuminate\\Http\\Request), Object(Closure))\n#28 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))\n#29 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Auth\\Middleware\\Authenticate.php(43): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}(Object(Illuminate\\Http\\Request))\n#30 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(163): Illuminate\\Auth\\Middleware\\Authenticate->handle(Object(Illuminate\\Http\\Request), Object(Closure))\n#31 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))\n#32 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Middleware\\VerifyCsrfToken.php(75): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}(Object(Illuminate\\Http\\Request))\n#33 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(163): Illuminate\\Foundation\\Http\\Middleware\\VerifyCsrfToken->handle(Object(Illuminate\\Http\\Request), Object(Closure))\n#34 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))\n#35 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\View\\Middleware\\ShareErrorsFromSession.php(49): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}(Object(Illuminate\\Http\\Request))\n#36 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(163): Illuminate\\View\\Middleware\\ShareErrorsFromSession->handle(Object(Illuminate\\Http\\Request), Object(Closure))\n#37 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))\n#38 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Session\\Middleware\\StartSession.php(63): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}(Object(Illuminate\\Http\\Request))\n#39 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(163): Illuminate\\Session\\Middleware\\StartSession->handle(Object(Illuminate\\Http\\Request), Object(Closure))\n#40 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))\n#41 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Cookie\\Middleware\\AddQueuedCookiesToResponse.php(37): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}(Object(Illuminate\\Http\\Request))\n#42 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(163): Illuminate\\Cookie\\Middleware\\AddQueuedCookiesToResponse->handle(Object(Illuminate\\Http\\Request), Object(Closure))\n#43 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))\n#44 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Cookie\\Middleware\\EncryptCookies.php(66): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}(Object(Illuminate\\Http\\Request))\n#45 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(163): Illuminate\\Cookie\\Middleware\\EncryptCookies->handle(Object(Illuminate\\Http\\Request), Object(Closure))\n#46 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))\n#47 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(104): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}(Object(Illuminate\\Http\\Request))\n#48 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Router.php(684): Illuminate\\Pipeline\\Pipeline->then(Object(Closure))\n#49 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Router.php(659): Illuminate\\Routing\\Router->runRouteWithinStack(Object(Illuminate\\Routing\\Route), Object(Illuminate\\Http\\Request))\n#50 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Router.php(625): Illuminate\\Routing\\Router->runRoute(Object(Illuminate\\Http\\Request), Object(Illuminate\\Routing\\Route))\n#51 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Router.php(614): Illuminate\\Routing\\Router->dispatchToRoute(Object(Illuminate\\Http\\Request))\n#52 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Kernel.php(176): Illuminate\\Routing\\Router->dispatch(Object(Illuminate\\Http\\Request))\n#53 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Pipeline.php(30): Illuminate\\Foundation\\Http\\Kernel->Illuminate\\Foundation\\Http\\{closure}(Object(Illuminate\\Http\\Request))\n#54 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\fideloper\\proxy\\src\\TrustProxies.php(57): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}(Object(Illuminate\\Http\\Request))\n#55 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(163): Fideloper\\Proxy\\TrustProxies->handle(Object(Illuminate\\Http\\Request), Object(Closure))\n#56 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))\n#57 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest.php(31): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}(Object(Illuminate\\Http\\Request))\n#58 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(163): Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest->handle(Object(Illuminate\\Http\\Request), Object(Closure))\n#59 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))\n#60 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest.php(31): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}(Object(Illuminate\\Http\\Request))\n#61 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(163): Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest->handle(Object(Illuminate\\Http\\Request), Object(Closure))\n#62 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))\n#63 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Middleware\\ValidatePostSize.php(27): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}(Object(Illuminate\\Http\\Request))\n#64 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(163): Illuminate\\Foundation\\Http\\Middleware\\ValidatePostSize->handle(Object(Illuminate\\Http\\Request), Object(Closure))\n#65 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))\n#66 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Middleware\\CheckForMaintenanceMode.php(62): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}(Object(Illuminate\\Http\\Request))\n#67 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(163): Illuminate\\Foundation\\Http\\Middleware\\CheckForMaintenanceMode->handle(Object(Illuminate\\Http\\Request), Object(Closure))\n#68 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Routing\\Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))\n#69 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Pipeline\\Pipeline.php(104): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}(Object(Illuminate\\Http\\Request))\n#70 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Kernel.php(151): Illuminate\\Pipeline\\Pipeline->then(Object(Closure))\n#71 C:\\xampp\\htdocs\\laravel_site_offline\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Http\\Kernel.php(116): Illuminate\\Foundation\\Http\\Kernel->sendRequestThroughRouter(Object(Illuminate\\Http\\Request))\n#72 C:\\xampp\\htdocs\\laravel_site_offline\\public\\index.php(55): Illuminate\\Foundation\\Http\\Kernel->handle(Object(Illuminate\\Http\\Request))\n#73 {main}"
```

**Comment by doubleprincez** [comment]
I've been able to trace the error down to .env......... paystack_payment_url ought to be  https://api.paystack.co but i was using my paying url i.e http://site.dev/payment/webhook
